### PR TITLE
feat(ButtonLink): scale chevron according to font size

### PR DESCRIPTION
**Pull request by marcoskolodny**
Font size = 10
image

Font size = 24
image

Font size = 72
image

### CODE REVIEW

**Review by yceballost — APPROVED** [review]

File: src/button.tsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ const renderButtonElement = ({
 
 const ButtonLinkChevron: React.FC = () => {
     return (
-        <svg width="8" height="20" viewBox="0 0 8 20">
+        <svg width="0.5em" height="1.25em" viewBox="0 0 8 20">
```
Comment (atabel) [inline src/button.tsx:84]: please use `pxToRem()` function

File: src/__screenshot_tests__/__image_snapshots__/title-screenshot-test-mobile_android-with-link-and-long-text-snap.png
Comment (atabel) [inline src/__screenshot_tests__/__image_snapshots__/title-screenshot-test-mobile_android-with-link-and-long-text-snap.png:1]: hmmm... this is weird, the size shouldn't change in the default case

Comment (marcoskolodny) [reply src/__screenshot_tests__/__image_snapshots__/title-screenshot-test-mobile_android-with-link-and-long-text-snap.png:1]: Actually, it's the expected behaviour: in mobile, Text2 has fontSize equal to 14px, not 16px

File: src/button.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/button.tsx:84]: Discussed offline. Using `rem` is not correct. We want the chevron to adapt to the button text's font size, and this size changes between devices (14px in mobile, 16 in desktop). For this reason, it's better to leave it with `em`.

**Pull request merged**